lib/client.js

```diff
@@ -660,13 +660,13 @@ Request.prototype.query = function(val){
  *
  * @param {String} field
  * @param {Blob|File} file
- * @param {String} filename
+ * @param {String|Object} options
  * @return {Request} for chaining
  * @api public
  */
 
-Request.prototype.attach = function(field, file, filename){
-  this._getFormData().append(field, file, filename || file.name);
+Request.prototype.attach = function(field, file, options){
+  this._getFormData().append(field, file, options || file.name);
```

Comment:
No, as the [third parameter is an object](https://github.com/form-data/form-data/blob/19748778e2cd05e2ddc95409fcbbf0461a868775/lib/form_data.js#L44), even though a string is accepted.